benchmarks/pubmedqa/pubmedqa.py

```diff
@@ -0,0 +1,58 @@
+"""
+PubMedQA: A Dataset for Biomedical Research Question Answering
+
+# inspect eval pubmedqa/pubmedqa.py
+"""
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import choice
+from inspect_ai.solver import multiple_choice
+
+TEMPLATE = r"""
+The following are multiple choice questions (with answers) about medical knowledge.
```

Comment:
This line implies that there're going to be multiple questions in this same template. Shouldn't this line be the system prompt instead?

Reply:
This was the prompt used in the original paper - the few-shot examples would follow this line, before the "Answer the following question qiven the context" below. When run zero-shot, the examples were excluded but this line didn't change.
I'm happy to either 
- move this line into the system prompt
- alter the prompt slightly to something like "The following is a multiple choice question about medical knowledge. Answer it given the context." or just "Answer the following multiple choice question about medical knowledge given the context".

Unsure how to trade off adherence to the original prompt vs readability/coherence here.

Reply:
If we aren't doing fewshot I think we should just strike that line (granted the original paper left it in, but I'd argue this was likely a programming oversight and we'd rather get to a correct prompt than adhere too closely to what was used). So it would be something like:

```python
TEMPLATE=r"""
Answer the following multiple choice question about medical knowledge given the context. 
The entire content of your response should be of the following format: 'ANSWER: $LETTER' 
(without quotes) where LETTER is one of {letters}.

{question}

{choices}
""".strip()
```